opus/import/populate_obs_instrument_VGISS_prof.py

```diff
@@ -0,0 +1,536 @@
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    inc = index_row['INCIDENCE_ANGLE']
+
+    return inc
+
+# Emission angle: the angle between the normal vector on the LIT side, to the
+# direction where outgoing photons to the observer. 0-90 when observer is at the
+# lit side of the ring, and 90-180 when it's at the dark side.
+def populate_obs_ring_geometry_VGISS_emission1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    ea = index_row['MINIMUM_EMISSION_ANGLE']
+
+    return ea
+
+def populate_obs_ring_geometry_VGISS_emission2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    ea = index_row['MAXIMUM_EMISSION_ANGLE']
+
+    return ea
+
+# North based ea: the angle between the normal vector on the NORTH side of the
+# ring, to the direction where outgoing photons to the observer. 0-90 when
+# observer is at the north side of the ring, and 90-180 when it's at the south
+# side.
+def populate_obs_ring_geometry_VGISS_north_based_emission1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    ea = index_row['MINIMUM_EMISSION_ANGLE']
+
+    return ea
+
+def populate_obs_ring_geometry_VGISS_north_based_emission2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    ea = index_row['MAXIMUM_EMISSION_ANGLE']
+
+    return ea
+
+# We set the center versions to be the same as the normal versions
+populate_obs_ring_geometry_VGISS_center_phase1_PROF = \
+    populate_obs_ring_geometry_VGISS_phase1_PROF
+populate_obs_ring_geometry_VGISS_center_phase2_PROF = \
+    populate_obs_ring_geometry_VGISS_phase2_PROF
+populate_obs_ring_geometry_VGISS_center_incidence1_PROF = \
+    populate_obs_ring_geometry_VGISS_incidence1_PROF
+populate_obs_ring_geometry_VGISS_center_incidence2_PROF = \
+    populate_obs_ring_geometry_VGISS_incidence2_PROF
+populate_obs_ring_geometry_VGISS_center_emission1_PROF = \
+    populate_obs_ring_geometry_VGISS_emission1_PROF
+populate_obs_ring_geometry_VGISS_center_emission2_PROF = \
+    populate_obs_ring_geometry_VGISS_emission2_PROF
+populate_obs_ring_geometry_VGISS_center_north_based_incidence1_PROF = \
+    populate_obs_ring_geometry_VGISS_north_based_incidence1_PROF
+populate_obs_ring_geometry_VGISS_center_north_based_incidence2_PROF = \
+    populate_obs_ring_geometry_VGISS_north_based_incidence2_PROF
+populate_obs_ring_geometry_VGISS_center_north_based_emission1_PROF = \
+    populate_obs_ring_geometry_VGISS_north_based_emission1_PROF
+populate_obs_ring_geometry_VGISS_center_north_based_emission2_PROF = \
+    populate_obs_ring_geometry_VGISS_north_based_emission2_PROF
+
+# Opening angle to observer: the angle between the ring surface to the direction
+# where outgoing photons to the observer. Positive if observer is at the north
+# side of the ring , negative if it's at the south side. In this case, observer
```

Comment:
Actually in this case the observer is on the south side. The proper thing to say is it's always 90 - north-based-emission-angle.

Reply:
done